packages/blockchain-wallet-v4-frontend/src/scenes/Register/template.tsx

```diff
@@ -224,7 +224,7 @@ const SignupCard = ({
             <InfoTitle color='grey800' size='18px' weight={600}>
               <FormattedMessage
                 id='scenes.register.walletcard.infotitle'
-                defaultMessage='Be your own bank.'
+                defaultMessage='Be Your Own Bank'
```

Comment:
need to also update the `id` if you want this change to take effect. eg =>  `id='scenes.register.walletcard.infotitleuppercase'`